index.js

```diff
@@ -1,86 +1 @@
-
-const findContentItem = require('./searches/findContentItem');
-const findWorkflowStep = require('./searches/findWorkflowStep');
-const findLanguage = require('./searches/findLanguage');
-const findAsset = require('./searches/findAsset');
-
-const deleteLanguageVariant = require('./actions/deleteLanguageVariant');
-const createContentItem = require('./actions/createContentItem');
-const updateLanguageVariant = require('./actions/updateLanguageVariant');
-const changeContentItemWorkflow = require('./actions/changeContentItemWorkflow');
-
-// We can roll up all our behaviors in an App.
-const App = {
-    // This is just shorthand to reference the installed dependencies you have. Zapier will
-    // need to know these before we can upload
-    version: require('./package.json').version,
-    platformVersion: require('zapier-platform-core').version,
-
-    authentication: authentication,
-
-    // beforeRequest & afterResponse are optional hooks into the provided HTTP client
-    beforeRequest: [],
-
-    afterResponse: [],
-
-    // If you want to define optional resources to simplify creation of triggers, searches, creates - do that here!
-    resources: {},
-
-    // If you want your trigger to show up, you better include it here!
-    triggers: {
-        [triggerWorkflowStatusChanged.key]: triggerWorkflowStatusChanged,
-        [triggerTaxonomyChanged.key]: triggerTaxonomyChanged,
-        [triggerItemPublishChanged.key]: triggerItemPublishChanged,
-        [triggerItemExistenceChanged.key]: triggerItemExistenceChanged,
-
-        // Lists for dropdowns
-        [getLinkedItems.key]: getLinkedItems,
-        [getContentTypes.key]: getContentTypes,
-        [getContentItems.key]: getContentItems,
-        [getWorkflowSteps.key]: getWorkflowSteps,
-        [getLanguages.key]: getLanguages,
-    },
-
-    // If you want your searches to show up, you better include it here!
-    searches: {
-        [findContentItem.key]: findContentItem,
-        [findWorkflowStep.key]: findWorkflowStep,
-        [findLanguage.key]: findLanguage,
-        [findAsset.key]: findAsset,
-    },
-
-    // If you want your creates to show up, you better include it here!
-    creates: {
-        [createContentItem.key]: createContentItem,
-        [changeContentItemWorkflow.key]: changeContentItemWorkflow,
-        [updateLanguageVariant.key]: updateLanguageVariant,
-        [deleteLanguageVariant.key]: deleteLanguageVariant,
-    },
-
-    searchOrCreates: {
-        find_item: {
-            search: 'find_item',
-            create: 'create_item',
-            key: 'find_item',
-            display: {
-                'description': 'Finds a content item matching the provided parameters. If more items match, it returns the first found item.',
-                'label': 'Find or create content item'
-            }
-        }
-    }
-};
-
-// Finally, export the app.
-module.exports = App;
+module.exports = require('./lib').default;
```

Comment:
This is from the official TS example https://github.com/zapier/zapier-platform/tree/master/example-apps/typescript. I take it this is a recommended approach to TS integrations. It would be possible to generate this file from a `.ts` file too, but I don't see any benefits in it and it would probably add some complications to the existing configuration.